crates/re_data_store/tests/snapshots/memory_test__scalars_on_one_timeline.snap

```diff
@@ -1,10 +1,9 @@
 ---
 source: crates/re_data_store/tests/memory_test.rs
-assertion_line: 96
 expression: "[format!(\"{NUM_SCALARS} scalars\"),\n        format!(\"{} in total\", re_format::format_bytes(total_mem_use as _)),\n        format!(\"{} per row\",\n            re_format::format_bytes(total_mem_use as f64 / NUM_SCALARS as\n            f64))]"
 ---
 [
     "1048576 scalars",
-    "936 MiB in total",
-    "936 B per row",
+    "912 MiB in total",
+    "912 B per row",
```

Comment:
That's one small step in the right direction 😆

Reply:
:disappointed: